oqs-template/generate.yml

```diff
@@ -526,18 +767,99 @@ sigs:
                           'pretty_name': 'RSA3072',
                           'oid': '1.3.9999.3.3',
                           'code_point': '0xfe0d'}]
+
+      -
+        name: 'falcon512WithSha256'
+        pretty_name: 'Falcon512-SHA256'
+        oqs_meth: 'OQS_SIG_alg_falcon_512'
+        oid: '1.3.6.1.4.1.18227.999.2.3.1.1'
+        digest: SHA256
+        code_point: '0xfec1'
+        supported_encodings: ['draft-uni-qsckeys-dilithium-00/sk-pk']
+        enable: true
+      
+      -
+        name: 'falcon512WithSha384'
+        pretty_name: 'Falcon512-SHA384'
+        oqs_meth: 'OQS_SIG_alg_falcon_512'
+        oid: '1.3.6.1.4.1.18227.999.2.3.2.1'
+        digest: SHA384
+        code_point: '0xfec2'
+        supported_encodings: ['draft-uni-qsckeys-dilithium-00/sk-pk']
+        enable: true
+
+      -
+        name: 'falcon512WithSha512'
+        pretty_name: 'Falcon512-SHA512'
+        oqs_meth: 'OQS_SIG_alg_falcon_512'
+        oid: '1.3.6.1.4.1.18227.999.2.3.3.1'
+        digest: SHA512
+        code_point: '0xfec3'
+        supported_encodings: ['draft-uni-qsckeys-dilithium-00/sk-pk']
+        enable: true
+      
+      -
+        name: 'falcon512WithSha3At256'
+        pretty_name: 'Falcon512-SHA3-256'
+        oqs_meth: 'OQS_SIG_alg_falcon_512'
+        oid: '1.3.6.1.4.1.18227.999.2.3.4.1'
+        digest: SHA3-256
+        code_point: '0xfec4'
+        supported_encodings: ['draft-uni-qsckeys-dilithium-00/sk-pk']
+        enable: true
+
+      -
+        name: 'falcon512WithSha3At384'
+        pretty_name: 'Falcon512-SHA3-384'
+        oqs_meth: 'OQS_SIG_alg_falcon_512'
+        oid: '1.3.6.1.4.1.18227.999.2.3.5.1'
+        digest: SHA3-384
+        code_point: '0xfec5'
+        supported_encodings: ['draft-uni-qsckeys-dilithium-00/sk-pk']
+        enable: true
+
+      -
+        name: 'falcon512WithSha3At512'
+        pretty_name: 'Falcon512-SHA3-512'
+        oqs_meth: 'OQS_SIG_alg_falcon_512'
+        oid: '1.3.6.1.4.1.18227.999.2.3.6.1'
+        digest: SHA3-512
+        code_point: '0xfec6'
+        supported_encodings: ['draft-uni-qsckeys-dilithium-00/sk-pk']
+        enable: true
+        
+      -
+        name: 'falcon512WithShake128'
+        pretty_name: 'Falcon512-SHAKE128'
+        oqs_meth: 'OQS_SIG_alg_falcon_512'
+        oid: '1.3.6.1.4.1.18227.999.2.3.7.1'
+        digest: SHAKE128
+        code_point: '0xfec6'
```

Comment:
This is a surefire way to cause TLS test problems: This code point is a duplicate. Please review all IDs _very_ carefully and re-test.

Reply:
Ahhhhh thank you so much for finding this!